portal/.stylelintrc.json

```diff
@@ -12,7 +12,8 @@
           "flexbox",
           "multicolumn",
           "css-nesting",
-          "css-overflow"
+          "css-overflow",
+          "css-display-contents"
```

Comment:
1. `overflow` is a commonly used property and all browsers supports it. What is not supported is `overflow: clip`. The rule seems does not check the value and report error for all `overflow` so I disabled the rule.
2. `display: contents` Is actually ok to use if we do not use it on a button. So I disabled the rule here. See https://caniuse.com/css-display-contents